src/NServiceBus.Core/Transports/Msmq/MsmqMessageSender.cs

```diff
@@ -50,7 +51,9 @@ public void Send(TransportMessage message, Address address)
                                 new MessageQueue(MsmqUtilities.GetReturnAddress(message.ReplyToAddress.ToString(),
                                                                                 address.ToString()));
 
-                        if (UnitOfWork.HasActiveTransaction())
+                        bool suppressNativeTransactions;
```

Comment:
I shouldn't write my questions in a hurry. @andreasohlund what I meant is, is there any scenario for removing timeouts where we want/need to rely on native transactions?
I was thinking about eliminating the pipeline flag (because that restricts us to 4.2+), but the only way I see we could do it is to change ISendMessagesInterface. Which is probably even a bigger mess.